state-chain/custom-rpc/src/lib.rs

```diff
@@ -252,6 +252,7 @@ pub enum RpcAccountInfo {
 	Broker {
 		flip_balance: NumberOrHex,
 		earned_fees: any::AssetMap<NumberOrHex>,
+		deposit_address: Option<String>,
```

Comment:
Let's make this clear and unambiguous: 

```suggestion
		btc_vault_deposit_address: Option<String>,
```